Apptainer python

loganharbour opened the pull request.

Commit:
Add Apptainer/Singularity definition files

Add Apptainer recipe/definition files.

Closes #22497

Commit:
Add empty Apptainer definition files

Add empty Apptainer recipe/definition files so that module_hash
functions correctly.

Closes #22505

Commit:
Rename MPI to MPICH, add ARCH

Apply ARCH identification.

We're using (going to use) MPICH in Apptainer. Conda is MPICH. Spack
is MPICH. Might as well just call the whole thing MPICH.

Naming it 'mpich' also helps with Conda package dependency tracking.
As that tool checks for moose/conda/<ZIP_KEY NAME> presence.

Commit:
Always forgetting the Unittests

Commit:
Simplify the table building process

Support printing tags with dependencies

Commit:
Need to reverse the list

Reverse the list when dealing with --dependencies

Commit:
Add missing tag only feature

Commit:
Add json/yaml output support

Commit:
Add oras format specific output

Default to printing HASH only. Add feature to print oras friendly URI.

Refactor meta dictionary a bit.

Commit:
Supply 'libmesh' as the default when appropriate

I don't think we will ever supply a built version/container of only
moose which others will 'depend on' in order to build their application.

When this scenario occurs, its best to instead supply libmesh as their
dependency.

loganharbour committed:
Add apptainer generator, rework module_hash

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1241802/) on 3840408 wanted to post the following:

The following commit(s) contain fixups:

- 38404087f05: fixup! 67e8b5fafbea6c5ea59f3c6f02089ef893f71836
- 67e8b5fafbe: fixup! c738c61dbf63ec883fab02c734bc0923a6eea2c8
- c738c61dbf6: fixup! 51c9be967b1a159e92cc3120dfda180887839033
- 51c9be967b1: fixup! 476b21c19367d23774cb884e4edb2cec4aee907f

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1241802/) on 3840408 wanted to post the following:

The following commit(s) contain fixups:

- 38404087f05: fixup! 67e8b5fafbea6c5ea59f3c6f02089ef893f71836
- 67e8b5fafbe: fixup! c738c61dbf63ec883fab02c734bc0923a6eea2c8
- c738c61dbf6: fixup! 51c9be967b1a159e92cc3120dfda180887839033
- 51c9be967b1: fixup! 476b21c19367d23774cb884e4edb2cec4aee907f

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1241891/) on 77f3d4a wanted to post the following:

The following commit(s) contain fixups:

- 77f3d4af67a: fixup! 38404087f0514cfa5a80cdd605001094ae974be6
- 38404087f05: fixup! 67e8b5fafbea6c5ea59f3c6f02089ef893f71836
- 67e8b5fafbe: fixup! c738c61dbf63ec883fab02c734bc0923a6eea2c8
- c738c61dbf6: fixup! 51c9be967b1a159e92cc3120dfda180887839033
- 51c9be967b1: fixup! 476b21c19367d23774cb884e4edb2cec4aee907f

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1241991/) on 46c5925 wanted to post the following:

The following commit(s) contain fixups:

- 46c59252786: fixup! 77f3d4af67a430c190bf9b868ef0e4a92c0cf71a
- 77f3d4af67a: fixup! 38404087f0514cfa5a80cdd605001094ae974be6
- 38404087f05: fixup! 67e8b5fafbea6c5ea59f3c6f02089ef893f71836
- 67e8b5fafbe: fixup! c738c61dbf63ec883fab02c734bc0923a6eea2c8
- c738c61dbf6: fixup! 51c9be967b1a159e92cc3120dfda180887839033
- 51c9be967b1: fixup! 476b21c19367d23774cb884e4edb2cec4aee907f

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242037/) on 3dab2fd wanted to post the following:

The following commit(s) contain fixups:

- 3dab2fd3290: fixup! 46c59252786dceedb8dd2e40667b25ab72e2a1ca
- 46c59252786: fixup! 77f3d4af67a430c190bf9b868ef0e4a92c0cf71a
- 77f3d4af67a: fixup! 38404087f0514cfa5a80cdd605001094ae974be6
- 38404087f05: fixup! 67e8b5fafbea6c5ea59f3c6f02089ef893f71836
- 67e8b5fafbe: fixup! c738c61dbf63ec883fab02c734bc0923a6eea2c8
- c738c61dbf6: fixup! 51c9be967b1a159e92cc3120dfda180887839033
- 51c9be967b1: fixup! 476b21c19367d23774cb884e4edb2cec4aee907f

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242076/) on d1b58ff wanted to post the following:

The following commit(s) contain fixups:

- d1b58fff3c3: fixup! 3dab2fd3290f3b0bb8104781b362838f1d0c3f1f
- 3dab2fd3290: fixup! 46c59252786dceedb8dd2e40667b25ab72e2a1ca
- 46c59252786: fixup! 77f3d4af67a430c190bf9b868ef0e4a92c0cf71a
- 77f3d4af67a: fixup! 38404087f0514cfa5a80cdd605001094ae974be6
- 38404087f05: fixup! 67e8b5fafbea6c5ea59f3c6f02089ef893f71836
- 67e8b5fafbe: fixup! c738c61dbf63ec883fab02c734bc0923a6eea2c8
- c738c61dbf6: fixup! 51c9be967b1a159e92cc3120dfda180887839033
- 51c9be967b1: fixup! 476b21c19367d23774cb884e4edb2cec4aee907f

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242300/) on a6295ef wanted to post the following:

The following commit(s) contain fixups:

- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242340/) on f586d79 wanted to post the following:

The following commit(s) contain fixups:

- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242570/) on 9d1eb84 wanted to post the following:

The following commit(s) contain fixups:

- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242609/) on 6b7d73b wanted to post the following:

The following commit(s) contain fixups:

- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242648/) on 078faea wanted to post the following:

The following commit(s) contain fixups:

- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242687/) on 2cfcf3d wanted to post the following:

The following commit(s) contain fixups:

- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242822/) on ba9188e wanted to post the following:

The following commit(s) contain fixups:

- ba9188e9429: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242900/) on 921ed98 wanted to post the following:

The following commit(s) contain fixups:

- 921ed98842f: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- de30783a506: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- ba9188e9429: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1242978/) on 06b2917 wanted to post the following:

The following commit(s) contain fixups:

- 06b2917634a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 921ed98842f: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- de30783a506: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- ba9188e9429: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1243168/) on d9714bd wanted to post the following:

The following commit(s) contain fixups:

- d9714bdee91: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 06b2917634a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 921ed98842f: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- de30783a506: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- ba9188e9429: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1243208/) on 2b83974 wanted to post the following:

The following commit(s) contain fixups:

- 2b83974462c: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- d9714bdee91: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 06b2917634a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 921ed98842f: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- de30783a506: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- ba9188e9429: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1243251/) on 056d366 wanted to post the following:

The following commit(s) contain fixups:

- 056d366e2ca: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2b83974462c: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- d9714bdee91: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 06b2917634a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 921ed98842f: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- de30783a506: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- ba9188e9429: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 2cfcf3d34b1: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 078faeac5bf: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 6b7d73bba64: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- 9d1eb8451ff: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- f586d79f67a: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e
- a6295ef58ea: fixup! 4c87961e05f0c8402d4bd8c2b1506745503bba8e

loganharbour committed:
Set changed_commit

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1249267/) on 4c89681 wanted to post the following:

The following commit(s) contain fixups:

- 4c89681c71b: fixup! 1eeaf0e9d2g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1249318/) on a65a195 wanted to post the following:

The following commit(s) contain fixups:

- a65a1959483: fixup! 1eeaf0e9d2g
- 4c89681c71b: fixup! 1eeaf0e9d2g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1249440/) on a9f2d8b wanted to post the following:

The following commit(s) contain fixups:

- a9f2d8bdf24: fixup! 1eeaf0e9d2g
- a65a1959483: fixup! 1eeaf0e9d2g
- 4c89681c71b: fixup! 1eeaf0e9d2g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1249517/) on 4f3a261 wanted to post the following:

The following commit(s) contain fixups:

- 4f3a261b7eb: fixup! 1eeaf0e9d2g
- a9f2d8bdf24: fixup! 1eeaf0e9d2g
- a65a1959483: fixup! 1eeaf0e9d2g
- 4c89681c71b: fixup! 1eeaf0e9d2g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1249571/) on e191e7b wanted to post the following:

The following commit(s) contain fixups:

- e191e7b80c7: fixup! fixup! 1eeaf0e9d2g
- 4f3a261b7eb: fixup! 1eeaf0e9d2g
- a9f2d8bdf24: fixup! 1eeaf0e9d2g
- a65a1959483: fixup! 1eeaf0e9d2g
- 4c89681c71b: fixup! 1eeaf0e9d2g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1250504/) on a3ccd22 wanted to post the following:

The following commit(s) contain fixups:

- a3ccd22d9ec: fixup! 1eeaf0e9d2g
- e191e7b80c7: fixup! fixup! 1eeaf0e9d2g
- 4f3a261b7eb: fixup! 1eeaf0e9d2g
- a9f2d8bdf24: fixup! 1eeaf0e9d2g
- a65a1959483: fixup! 1eeaf0e9d2g
- 4c89681c71b: fixup! 1eeaf0e9d2g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1251400/) on 73d3194 wanted to post the following:

The following commit(s) contain fixups:

- 73d31945c22: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- bf7b94e6d4d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1251478/) on a5e5d87 wanted to post the following:

The following commit(s) contain fixups:

- a5e5d870ada: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 3ff6806bd2d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 73d31945c22: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- bf7b94e6d4d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125

permcody committed:
Attempting to add an install_python target for docs

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1251681/) on 9a8f3f2 wanted to post the following:

The following commit(s) contain fixups:

- 9a8f3f2934f: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- a5e5d870ada: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 3ff6806bd2d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 73d31945c22: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- bf7b94e6d4d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1252488/) on 9369ec0 wanted to post the following:

The following commit(s) contain fixups:

- 9369ec0b75c: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 8a84898539a: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 9a8f3f2934f: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- a5e5d870ada: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 3ff6806bd2d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 73d31945c22: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- bf7b94e6d4d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1252622/) on 083d65c wanted to post the following:

The following commit(s) contain fixups:

- 083d65c91db: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 9369ec0b75c: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 8a84898539a: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 9a8f3f2934f: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- a5e5d870ada: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 3ff6806bd2d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 73d31945c22: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- bf7b94e6d4d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1252926/) on 36b2b72 wanted to post the following:

The following commit(s) contain fixups:

- 36b2b725bbf: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 083d65c91db: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 9369ec0b75c: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 8a84898539a: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 9a8f3f2934f: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- a5e5d870ada: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 3ff6806bd2d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- 73d31945c22: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125
- bf7b94e6d4d: fixup! e5f9f3b4f548404b6aff99b8c3d4b0548e7c1125

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1252961/) on 36b2b72 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>9cdaac</th>
      <th colspan=4>#22778 36b2b7</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>84.40%</td>
      <td>84.39%</td>
      <td>-0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>80707</td>
      <td>80703</td>
      <td>-4</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>14920</td>
      <td>14924</td>
      <td>+4</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/22778/coverage_diff/framework)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/22778/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/22778/coverage/xfem)

</details>



This comment will be updated on new commits.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254707/) on 2e0599e wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254873/) on 22776fd wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254955/) on 1535783 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254955/) on 1535783 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254955/) on 1535783 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Add setuptools to build dep

Add setuptools to build dep, add python to run (to generate the py prefix)

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254997/) on 8b859bf wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254997/) on 8b859bf wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1254997/) on 8b859bf wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Fix broken yaml syntax

Remove extra space. Update dependency header help.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255163/) on dc0b0d6 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255163/) on dc0b0d6 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255163/) on dc0b0d6 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255243/) on f7f4e93 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255243/) on f7f4e93 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
remove openssl constraints

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255452/) on c8251b8 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255452/) on c8251b8 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255452/) on c8251b8 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Also set MOOSE_TEST_TOOLS_VERSION

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255539/) on 54a53f6 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Cleanup app definition

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255718/) on 93b338d wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255718/) on 93b338d wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Separate pkgs from moose-tools to moose-test-tools

Help shrink the size of Apptainer containers that do not need everything
present in moose-tools. The two packages will still be closely tied to
eachother. This should have no effect on End Users. They will still end up installing
the same packages.

Remove pinning of openssl where applicable.

loganharbour committed:
Move installs to /opt

loganharbour committed:
Add more packages to mpich installs

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255949/) on 35a3a0e wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255949/) on 35a3a0e wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255989/) on a97cf1c wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1255989/) on a97cf1c wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Remove development MOOSE_DIR

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256146/) on f1ecadb wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256146/) on f1ecadb wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Remove X11

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256186/) on 9667049 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256186/) on 9667049 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256186/) on 9667049 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Add libmesh-vtk meta.yaml to libmesh tracking

loganharbour committed:
More /usr/local -> /opt changes

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256592/) on 9420ef7 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256592/) on 9420ef7 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256592/) on 9420ef7 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Pass entire yaml load as meta

Add VTK build config to libmesh apptainer

Commit:
Adding VTK to libmesh container def

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256846/) on a244eb7 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256846/) on a244eb7 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256846/) on a244eb7 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Add VTK vars to requirements section

Generalize the jinja2/bash variables

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256886/) on c67cbf1 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256886/) on c67cbf1 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256886/) on c67cbf1 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Fix extra argument mistake

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256966/) on f204ebc wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1256966/) on f204ebc wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Export necessary VTK variables

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1257788/) on a1a137c wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1257788/) on a1a137c wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Add unzip to mpich container

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259059/) on 6acc709 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259059/) on 6acc709 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Add apptainer_generator as a dependency to all the things

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259327/) on de26b09 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259327/) on de26b09 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259327/) on de26b09 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259367/) on 9e5d115 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259367/) on 9e5d115 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259618/) on c316a6f wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259618/) on c316a6f wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Add quotes around MOOSE_DOCS_FLAGS

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259779/) on 0bc44ef wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259779/) on 0bc44ef wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259779/) on 0bc44ef wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259857/) on 290a84e wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259857/) on 290a84e wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259857/) on 290a84e wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Add packages needed for documentation to app install

loganharbour committed:
Remove now unnecessary comment

loganharbour committed:
Add trailing slashes for copies, ensure directories exist before copy

loganharbour reviewed (commented):
Approving changes that are not my own per independent review.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259909/) on 06696ed wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259909/) on 06696ed wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259909/) on 06696ed wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Remove extraneous entry

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259949/) on 1837ab8 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259949/) on 1837ab8 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1259949/) on 1837ab8 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Fix meta yaml skip

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260026/) on 034acd5 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260026/) on 034acd5 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260026/) on 034acd5 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Also need petsc and libmesh for documentation

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260137/) on 1f3a2e4 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260137/) on 1f3a2e4 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour committed:
Get submodule hashes for MOOSE submodules from MOOSE_DIR

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260277/) on e14d51f wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260277/) on e14d51f wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260277/) on e14d51f wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

cticenhour commented on apptainer/petsc.def:
```diff
@@ -0,0 +1,58 @@
+
+    # Test PETSc
+    cd petsc
+    make SLEPC_DIR=${PETSC_DIR} PETSC_DIR=${PETSC_DIR} PETSC_ARCH= check
```
No arch?

loganharbour replied:
Nope!

loganharbour committed:
Apply suggestions from code review

Co-authored-by: Casey Icenhour <cticenho@ncsu.edu>

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260639/) on bd44851 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260639/) on bd44851 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1260639/) on bd44851 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops
